_posts/2016-06-20-user-testing.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "Usability Testing - Our Process and Favorite Practices"
+date: 2016-06-20
+author: Anne Tomasevich
+tags: user-experience best-practices
+summary: Our approach to designing, conducting, and learning from user testing and other analyses.
+featured_image: "/assets/img/blog/usability-testing.png"
+featured_image_alt: "Two people comparing notes while using computers"
+---
+
+One of my favorite lines about usability testing comes from Jakob Nielson of the Nielson Norman Group in his article [Why You Only Need to Test with 5 Users](https://www.nngroup.com/articles/why-you-only-need-to-test-with-5-users/), regarding a graph of number of user test participants versus percentage of usability problems found:
+
+> The most striking truth of the curve is that zero users give zero insights.
+
+This, along with [other persuasive evidence](https://articles.uie.com/three_hund_million_button/), is a fantastic reminder that we can learn so much about what's right and what's wrong with our websites simply by watching people use them.
+
+At Savas Labs we've honed our approach to usability testing to balance efficiency, budget, and effectiveness. I'd like to share our process, some of my favorite resources, and some lessons we've learned along the way.
+
+## Our Process
+
+### Determine goals and user personas
```

Comment:
@AnneTee is it worth mentioning anywhere of the idea of putting on our consultative 🎩  and helping identify important users that sometimes clients forget, like themselves (or whoever administers the site)